modules/platforms/dotnet/Apache.Ignite.Core/Binary/BinaryBasicNameMapper.cs

```diff
@@ -45,6 +46,20 @@ public class BinaryBasicNameMapper : IBinaryNameMapper
         /// </summary>
         public bool IsSimpleName { get; set; }
 
+        /// <summary>
+        /// Domain to be added as a first part of Java type name.
+        /// Java and .Net assumes usage of different naming conventions.
```

Comment:
Here and below - it is spelled `.NET`, all upper case.

Reply:
Fixed.